/tmp/urllib3/dummyserver/server.py:46:18: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/src/urllib3/contrib/pyopenssl.py:84:5: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/src/urllib3/contrib/pyopenssl.py:85:5: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/src/urllib3/util/retry.py:301:30: DUO102 insecure use of "random" module, prefer "random.SystemRandom"
/tmp/urllib3/test/conftest.py:307:31: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/test_exceptions.py:44:18: DUO103 insecure use of "pickle" or "cPickle"
/tmp/urllib3/test/test_ssl.py:173:32: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/test_util.py:643:13: DUO131 "urllib3" warnings disabled, insecure connections possible
/tmp/urllib3/test/test_util.py:975:14: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/test_util.py:975:29: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/test_util.py:992:14: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/test_util.py:992:35: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_https.py:372:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:375:23: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_https.py:390:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:525:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:539:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:595:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:608:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_https.py:768:46: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_https.py:833:31: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_no_ssl.py:26:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
/tmp/urllib3/test/with_dummyserver/test_proxy_poolmanager.py:649:24: DUO130 insecure use of "hashlib" module
/tmp/urllib3/test/with_dummyserver/test_socketlevel.py:143:33: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_socketlevel.py:144:24: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_socketlevel.py:204:25: DUO122 insecure "ssl" module attribute use
/tmp/urllib3/test/with_dummyserver/test_socketlevel.py:1257:14: DUO132 "urllib3" certificate verification disabled, insecure connections possible
